docs/api/Components.md

```diff
@@ -0,0 +1,101 @@
+
+ * Pass it through props to every child
+ * Use React's context
+ 
+We recommend using React's context, since it will implicitly handle propagation as long as the controller view registers its `contextTypes`. We provide a couple of helpers to make this easier: 
+
+## FluxibleComponent
+
+The `FluxibleComponent` is a wrapper component that will provide all of its children with access to the Fluxible component
+context via React's `childContextTypes` and `getChildContext`. This should be used to wrap your top level component. It provides access to the methods on the [component context](#Component Context).
+
+ You can get access to these methods by setting the correct `contextTypes` within your component or including the [`FluxibleMixin`](FluxibleMixin.md) which will add them for you.
+
+### Usage
+
+If you have a component that needs access to the [`ComponentContext`](ComponentContext.md) methods:
+
+ ```js
+var Component = React.createClass({
+    contextTypes: {
+        getStore: React.PropTypes.func.isRequired,
+        executeAction: React.PropTypes.func.isRequired
+    },
+    getInitialState: function () {
+        return this.getStore(FooStore).getState();
+    }
+});
+
+you can wrap the component with `FluxibleComponent` to provide the correct context:
+
+```js
+var FluxibleComponent = require('fluxible').FluxibleComponent;
+var html = React.renderToString(
+    <FluxibleComponent context={context.getComponentContext()}>
+        <Component />
+    </FluxibleComponent>
+);
+```
+
+ If you are using [`FluxibleContext.createElement()`](FluxibleContext.md#createElement) this will happen for you automatically:
+
+```js
+var html = React.renderToString(context.createElement());
+```
+
+## FluxibleMixin
+
+The mixin (accessible via `require('fluxible').FluxibleMixin`) will add the `contextTypes` `getStore` and `executeAction`
+to your component.
+
+The mixin can also be used to statically list store dependencies and listen to them automatically in componentDidMount. This is done by adding a static property `storeListeners` in your component.
+
+You can do this with an array, which will default all store listeners to call the `onChange` method:
+
+```js
+var FluxibleMixin = require('fluxible').FluxibleMixin;
+var MockStore = require('./stores/MockStore'); // Your store
+var Component = React.createClass({
+    mixins: [FluxibleMixin],
+    statics: {
+        storeListeners: [MockStore]
+    },
+    onChange: function () {
+        this.setState(this.getStore(MockStore).getState());
+    },
+});
+```
+
+Or you can be more explicit with which function to call for each store by using a hash:
+
+```js
+var FluxibleMixin = require('fluxible').FluxibleMixin;
+var MockStore = require('./stores/MockStore'); // Your store
+var Component = React.createClass({
+    mixins: [FluxibleMixin],
+    statics: {
+        storeListeners: {
+            onMockStoreChange: [MockStore]
```

Comment:
Rename to something besides `MockStore` such as `FooStore`. And let's also add another store demonstrating multiple stores' change events being handled by different functions (`BarStore` => `onBarStoreChange`).